packages/sdk/README.md

```diff
@@ -38,6 +38,34 @@ The current regions are: `eu` and `us`.
 
 The timeout option is used to set the timeout for the feature flag resolve network request to the Confidence backend. When the timeout is reached, default values will be returned.
 
+### Logging
+
+During your integration and when debugging, you can get helpful logging information by defining a `logger` when creating the Confidence instance. The `Logger` is an interface for you to implement, or you can also pass `console`.
+
+```ts
+import { Confidence } from '@spotify-confidence/sdk';
+
+const myLogger = {
+  info: _ => {
+    // don't log anything
+  },
```

Comment:
This should instead just be removed. I.e. If you don't want to log info, don't provide info.
```suggestion
```